src/components/pengestopp/PengestoppHistorikk.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+import { Element, Undertittel } from "nav-frontend-typografi";
+import {
+  Arbeidsgiver,
+  StatusEndring,
+} from "../../data/pengestopp/types/FlaggPerson";
+import {
+  sykmeldingerToArbeidsgiver,
+  uniqueArbeidsgivere,
+} from "../../utils/pengestoppUtils";
+import { SykmeldingOldFormat } from "../../data/sykmelding/types/SykmeldingOldFormat";
+import { sykepengestoppArsakTekstListe } from "./PengestoppModal";
+import Panel from "nav-frontend-paneler";
+import styled from "styled-components";
+import { texts } from "./Pengestopp";
+
+interface IPengestoppDropdown {
+  statusEndringList: StatusEndring[];
+  sykmeldinger: SykmeldingOldFormat[];
+}
+
+const navLysBla = "#e0f5fb";
+const navLysBlaDarken = "#5690a2";
```

Comment:
Om disse er tenkt å kunne gjenbrukes, så burde de ligge en en egen fil for dette.